resources/artifact-ignores.properties

```
@@ -206,6 +206,7 @@ sms-notification                 # renamed to sms
 sonargraph-plugin = https://git.io/JJ5h1
 # deprecated (see Sonatype CLM: https://wiki.jenkins-ci.org/x/iYDPAw)
 sonatype-ci = https://wiki.jenkins.io/x/iYDPAw
+squash4jenkins                   # removal due to security issue JIRA SECURITY-2525
```

```suggestion
squashtm-publisher                   # removal due to security issue JIRA SECURITY-2525
```

From https://github.com/jenkinsci/squashtm-publisher-plugin/blob/5641bf160f66be9a93bca2f7f1555c3ae3de6d08/build.gradle#L22